tracer/src/Datadog.Trace.Coverage.collector/AssemblyProcessor.cs

```diff
@@ -109,11 +124,20 @@ public void Process()
                     }
                     else if (tracerTarget == TracerTarget.Net461)
                     {
-                        _logger.Debug($"Assembly: {FilePath}, is a net461 signed assembly, a .snk file is required ({Configuration.ConfigurationKeys.CIVisibility.CodeCoverageSnkFile} environment variable).");
+                        _logger.Warning($"Assembly: {FilePath}, is a net461 signed assembly, a .snk file is required ({Configuration.ConfigurationKeys.CIVisibility.CodeCoverageSnkFile} environment variable).");
+                        return;
+                    }
+                    else if (hasInternalsVisibleAttribute)
+                    {
+                        _logger.Warning($"Assembly: {FilePath}, is a signed assembly with the InternalsVisibleTo attribute. A .snk file is required ({Configuration.ConfigurationKeys.CIVisibility.CodeCoverageSnkFile} environment variable).");
                         return;
```

Comment:
This isn't _guaranteed_ to be an issue is it? 🤔  I mean, it _probably_ will, but not guaranteed? Agree we should definitely add a warning, but are we sure we want to automatically bail too?